bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -35,226 +35,229 @@ Location,Garden,Outdoor,Garden,Gardens,
-Equipment,Shower,Wellness,Shower,,
-Equipment,SwimmingPool,Wellness,Swimming Pool,"Swimming Pool, Pool",
-Equipment,WhiteGood,,White Good,White Goods,
-Equipment,AirFryer,WhiteGood,Air Fryer,,
-Equipment,CoffeeMaker,WhiteGood,Coffee Maker,"Coffee Makers, Coffee Machine, Coffee Machines",
-Equipment,Cooktop,WhiteGood,Cooktop,Hob,
-Equipment,Dishwasher,WhiteGood,Dishwasher,Dishwashers,
-Equipment,Dryer,WhiteGood,Dryer,"Dryers, Tumble Dryer, Tumble Dryers",
-Equipment,FoodProcessor,WhiteGood,Food Processor,,
-Equipment,Freezer,WhiteGood,Freezer,Freezers,
-Equipment,Fryer,WhiteGood,Fryer,,
-Equipment,IceMaker,WhiteGood,Ice Maker,,
-Equipment,Microwave,WhiteGood,Microwave,,
-Equipment,Mixer,WhiteGood,Mixer,,
-Equipment,Oven,WhiteGood,Oven,Ovens,
-Equipment,Range,WhiteGood,Range,,
-Equipment,Refrigerator,WhiteGood,Refrigerator,Refrigerators,
-Equipment,Toaster,WhiteGood,Toaster,Toaster Oven,
-Equipment,WashingMachine,WhiteGood,Washing Machine,Washing Machines,
-Equipment,Window,,Window,Windows,
-Equipment,WindowCovering,,Window Covering,,
-Equipment,Blinds,WindowCovering,Blinds,"Rollershutter, Rollershutters, Roller shutter, Roller shutters, Shutter, Shutters",
-Equipment,Drapes,WindowCovering,Drapes,Curtains,
-Equipment,Zone,,Zone,,
-Equipment,AlarmZone,Zone,Alarm Zone,,
+Property,Brightness,,Brightness,,Brightness of a light source
+Property,Channel,,Channel,,Radio or TV channel selection
+Property,Color,,Color,,Color of a light source
+Property,ColorTemperature,,Color Temperature,,Color temperature of a light source
+Property,Current,,Current,,Electric current
+Property,Duration,,Duration,,Time duration
+Property,Enabled,,Enabled,,Enabled state
+Property,Energy,,Energy,,Energy consumption or production
+Property,Frequency,,Frequency,,Electric frequency
+Property,Gas,,Gas,,Gas measurement
+Property,Heating,,Heating,,Related to heating
+Property,Humidity,,Humidity,Moisture,Humidity value
+Property,Illuminance,,Illuminance,,Illuminance value
+Property,Info,,Information,,General information
+Property,Level,,Level,,Fluid level
+Property,Light,,Light,"Lights, Lighting",Control or status of a light
+Property,LowBattery,,Low Battery,,Low battery indication
+Property,MediaControl,,Media Control,,Audio- or visual- media control
+Property,Mode,,Mode,,Operating mode
+Property,Moisture,,Moisture,,Moisture value
+Property,Motion,,Motion,,Motion detected
+Property,Noise,,Noise,,Noise measurement
+Property,Oil,,Oil,,Oil measurement
+Property,Opening,,Opening,,Opening of a door or window etc.
+Property,OpenLevel,Opening,Open Level,,Open position (analog) of a door or window etc.
+Property,OpenState,Opening,Open State,Open Closed,Open state (digital) of a door or window etc.
+Property,Position,,Position,,Position measurement or control
+Property,GeoLocation,Position,Geo Location,,Geo location coordinate
+Property,Power,,Power,,Power measurement
+Property,Presence,,Presence,,Presence detection
+Property,Pressure,,Pressure,,Fluid pressure measurement
```

Comment:
```suggestion
Property,Pressure,,Pressure,,
```
May apply to other than fluid, e.g. barometric pressure, bed pressure sensor, etc.